app/views/organisations/_show_organisation.html.erb

```diff
@@ -130,6 +130,21 @@
   <% end %>
 <% end %>
 
+<% if @show.high_profile_groups[:items] %>
+  <div class="grid-row organisations__section">
+    <div class="column-two-thirds">
+      <div class="organisations__brand-border-top brand--<%= @organisation.brand %> brand__border-color">
```

Comment:
I was putting this class on the wrapping div because Alex wanted full width top borders but 2/3 width title text. Since this whole section is already wrapped in a 2/3 column you can remove this div entirely and call the branding option on the heading component instead i.e. `brand: @organisation.brand`